### DISCUSSION TITLE
Separator-style not working for me, both none and solid give the default dashed version.

### DISCUSSION THREAD

**Discussion by nnuel** [davatorium/rofi#1568] [Theming]
It says it uses none, dash, solid from Xresources, does this mean it doesn't work under wayland?

**Comment by nnuel** [comment]
Nevermind, got it to work with `listview {border: 0px;}'. Took me a while to figure out.

**Reply by DaveDavenport** [reply]
next time please provide rofi version and the output of `rofi -dump-theme`.  It is hard to help without them.
